mobile-ui/src/main/java/org/buffer/android/boilerplate/ui/browse/BrowseAdapter.kt

```diff
@@ -9,20 +10,19 @@ import android.widget.TextView
 import com.bumptech.glide.Glide
 import com.bumptech.glide.request.RequestOptions
 import org.buffer.android.boilerplate.ui.R
-import org.buffer.android.boilerplate.ui.model.BufferooViewModel
+import org.buffer.android.boilerplate.ui.model.ArticleViewModel
 import javax.inject.Inject
 
-class BrowseAdapter @Inject constructor(): RecyclerView.Adapter<BrowseAdapter.ViewHolder>() {
+class BrowseAdapter @Inject constructor() : RecyclerView.Adapter<BrowseAdapter.ViewHolder>() {
 
-    var bufferoos: List<BufferooViewModel> = arrayListOf()
+    var articles: List<ArticleViewModel> = arrayListOf()
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        val bufferoo = bufferoos[position]
-        holder.nameText.text = bufferoo.name
-        holder.titleText.text = bufferoo.title
-
+        val article = articles[position]
+        holder.nameText.text = article.author
+        holder.titleText.text = article.title
         Glide.with(holder.itemView.context)
```

Comment:
This sample of using glide might also needs to be optimized, see here for details: https://stackoverflow.com/a/32887693/3763032